### ISSUE TITLE
assertion functions

### ISSUE THREAD

**Issue by ivan108** [UCSF-Costello-Lab/LG3_Pipeline#111]
I see that all assertion functions are silent, only reporting on errors. I wonder if it can useful also to report positive tests? May be in debug mode?

**Comment by HenrikBengtsson** [comment]
Yes, this could be added to a debug mode, e.g.
```r
debug "this and that"
```
where `debug()` outputs nothing if `LG3_DEBUG != true`.

**Comment by ivan108** [comment]
Could you add another assertion "assert_link_exists" to test the existence of symbolic links?
Thanks!

**Comment by HenrikBengtsson** [comment]
Can you clarify what such a `assert_link_exists()` function should do?  Make sure a file is a link and not an actual file?

**Comment by ivan108** [comment]
Yes, the "file" exists and is a link, and not broken. Some googling gave me this

[[ -L ${my_link} ]] && [[ -e ${my_link} ]]

**Comment by ivan108** [comment]
Added assert_link_exists()
